packages/gatsby-typegen-react-docgen/src/gatsby-node.js

```diff
@@ -0,0 +1,159 @@
+const {
+  GraphQLBoolean,
+  GraphQLObjectType,
+  GraphQLList,
+  GraphQLString,
+} = require(`graphql`)
+const path = require(`path`)
+const crypto = require(`crypto`)
+const GraphQLJSON = require(`graphql-type-json`)
+const metadata = require(`react-docgen`)
+
+const { parseDoclets, applyPropDoclets } = require(`./Doclets`)
+
+const digest = str => crypto.createHash(`md5`).update(str).digest(`hex`)
+
+function resolve(field) {
+  return obj => JSON.parse(obj.content)[field]
+}
+
+const descId = parent => `${parent.id}--ComponentMetadataDescription`
+
+function createMarkdownNode(node, boundActionCreators) {
+  if (!node.description) return
+  const { createNode, updateNode } = boundActionCreators
+
+  const markdownNode = {
+    id: descId(node),
+    type: `MarkdownRemark`,
+    mediaType: `text/x-markdown`,
+    parent: node.id,
+    content: node.description,
+    contentDigest: digest(node.description),
+    children: [],
+  }
```

Comment:
i probably don't actually mind the extra node tooo much but it doesn't add a ton of value, and yeah not helpful as a toplevel query